english/count-the-marshmellows/eat-all-the-marshmellows/get-breakfast/get-breakfast.md

```diff
@@ -1,7 +1,14 @@
-HECK YES. You found the best ending. (Breakfast is always the best ending.)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.